corehq/apps/accounting/migrations/0027_auto_20180509_1857.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-05-09 18:57
+from __future__ import unicode_literals
+from __future__ import absolute_import
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('accounting', '0026_auto_20180508_1956'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='billingrecord',
+            name='emailed_to',
+        ),
+        migrations.RemoveField(
+            model_name='wirebillingrecord',
+            name='emailed_to',
+        ),
+    ]
```

Comment:
You might want to put the remove operation in a separate PR to be merged later so that this is reversible if need be.